clang/test/dpct/math/cuda-math-extension.cu

```diff
@@ -181,6 +181,14 @@ __global__ void kernelFuncFloat(float *deviceArrayFloat) {
   f2 = __fmaf_ru(f0, f1, f2);
   // CHECK: f2 = sycl::ext::intel::math::fmaf_rz(f0, f1, f2);
   f2 = __fmaf_rz(f0, f1, f2);
+  // CHECK: f2 = sycl::ext::intel::math::fmaf_rd(f0, f1, f2);
```

Comment:
emit warning if there is diff on behavior.

Reply:
The flush-to-zero behavior can only be controlled by the compiler option in SYCL. Please verify the correctness of the migrated code.